2.7/s2i/bin/run

```diff
@@ -35,14 +35,34 @@ function get_default_web_concurrency() {
   echo $default
 }
 
+function maybe_run_in_init_wrapper() {
+  if [[ -z "$ENABLE_INIT_WRAPPER" ]]; then
+    exec $@
+  else
+    local wrapper=`mktemp -p .`
```

Comment:
I would prefer to remove wrapper file after tests or create it under `/tmp/`. Any reason why to create it in tests directory?

Reply:
I'm not sure how this relates to tests... You create the wrapper when starting the container and the only point when removing the wrapper would make sense is when the container is shutting down, at which point you don't care that there's a leftover file, do you?

Reply:
Oh, my fault. Sorry, you are right.